plugins/jukealert-paper/src/main/java/com/untamedears/jukealert/gui/SnitchOverviewGUI.java

```diff
@@ -29,38 +38,39 @@ public SnitchOverviewGUI(Player player, List<Snitch> snitches, String title, boo
 		this.canShowDetails = canShowDetails;
 	}
 
-	private List<IClickable> constructSnitchClickables() {
+	private List<IClickable> constructSnitchClickables(int start, int offset) {
 		final List<IClickable> clickables = new LinkedList<>();
-		for (final Snitch snitch : this.snitches) {
+		for (int i = start; i <= Math.min(snitches.size() - 1, start + offset); i++) {
+			Snitch snitch = snitches.get(i);
 			// Base the snitch icon on the snitch type
 			final var icon = snitch.getType().getItem().clone();
 			ItemUtils.handleItemMeta(icon, (ItemMeta meta) -> {
-				meta.setDisplayName(ChatColor.GOLD + snitch.getName());
+				meta.displayName(Component.text(snitch.getName(), NamedTextColor.GOLD));
 				final var location = snitch.getLocation();
-				MetaUtils.addLore(meta, ChatColor.AQUA + "Location: "
-						+ ChatColor.WHITE + location.getWorld().getName() + " "
-						+ ChatColor.RED + location.getBlockX()
-						+ ChatColor.WHITE + ", "
-						+ ChatColor.GREEN + location.getBlockY()
-						+ ChatColor.WHITE + ", "
-						+ ChatColor.BLUE + location.getBlockZ());
-				MetaUtils.addLore(meta, ChatColor.YELLOW + "Group: " + snitch.getGroup().getName());
+				List<Component> lore = new ArrayList<>();
+				lore.add(ChatUtils.newComponent("Location: ").color(NamedTextColor.AQUA)
+						.append(Component.text(location.getWorld().getName() + " ").color(NamedTextColor.WHITE))
+						.append(Component.text(location.getBlockX()).color(NamedTextColor.RED))
+						.append(Component.text(", ").color(NamedTextColor.WHITE))
+						.append(Component.text(location.getBlockY()).color(NamedTextColor.GREEN))
+						.append(Component.text(", ").color(NamedTextColor.WHITE))
+						.append(Component.text(location.getBlockZ()).color(NamedTextColor.BLUE)));
+				lore.add(ChatUtils.newComponent("Group: " + snitch.getGroup().getName()).color(NamedTextColor.YELLOW));
```

Comment:
You should do the same with all of these components too :]

Reply:
Nah